lib/theme.json

```diff
@@ -352,6 +352,21 @@
 							}
 						}
 					]
+				},
+				"grid": {
+					"name": "grid",
+					"slug": "grid",
+					"className": "is-layout-grid",
+					"displayMode": "grid",
+					"baseStyles": [],
```

Comment:
Looks like we might need to include the `margin: 0` rule from `baseStyles` used by the flex layout, too. Otherwise the user agent stylesheet margin rules for e.g. `p` tags apply, affecting the gap:

<img width="831" alt="image" src="https://user-images.githubusercontent.com/14988353/224862473-cd0a776f-657b-42f5-ad5d-524d0a55af46.png">